diff --git a/node_modules/@dpc-sdp/ripple-search/SearchResult.vue b/node_modules/@dpc-sdp/ripple-search/SearchResult.vue
index 5825769..4013029 100644
--- a/node_modules/@dpc-sdp/ripple-search/SearchResult.vue
+++ b/node_modules/@dpc-sdp/ripple-search/SearchResult.vue
@@ -8,7 +8,7 @@
       <span class="rpl-search-result__date">{{ formatDate(date, 'MMM D, YYYY') }}</span>
       <span>{{ description }}</span>
     </p>
-    <ul class="rpl-search-result__terms">
+    <ul class="rpl-search-result__terms" v-if="tags">
       <li class="rpl-search-result__term" v-for="(tag, index) in tags" :key="index">
         <rpl-meta-tag :linkText="tag.linkText" :linkUrl="tag.linkUrl" />
       </li>
diff --git a/node_modules/@dpc-sdp/ripple-search/SearchResultsLayout.vue b/node_modules/@dpc-sdp/ripple-search/SearchResultsLayout.vue
index cde18c0..3b5122c 100644
--- a/node_modules/@dpc-sdp/ripple-search/SearchResultsLayout.vue
+++ b/node_modules/@dpc-sdp/ripple-search/SearchResultsLayout.vue
@@ -5,7 +5,7 @@
     </div>
     <template v-else>
       <div class="rpl-search-results-layout__header">
-        <div class="rpl-search-results-layout__info" v-if="range && count">
+        <div class="rpl-search-results-layout__info" v-if="(range && count) || !!$slots.count">
           <slot name="count">
             Displaying {{ range }} of {{ count }} results
           </slot>
