diff --git a/node_modules/@dpc-sdp/ripple-form/Select.vue b/node_modules/@dpc-sdp/ripple-form/Select.vue
index 467e169..75e3182 100644
--- a/node_modules/@dpc-sdp/ripple-form/Select.vue
+++ b/node_modules/@dpc-sdp/ripple-form/Select.vue
@@ -376,7 +376,7 @@ $rpl-select-checkbox-margin: 0 !default;
     outline: 0;
     position: absolute;
     top: -1px;
-    z-index: 1;
+    z-index: 2;
     background: $rpl-form-element-bg-color;
     left: -1px;
     width: 100%;
diff --git a/node_modules/@dpc-sdp/ripple-form/index.vue b/node_modules/@dpc-sdp/ripple-form/index.vue
index c2ec90e..bdc903a 100644
--- a/node_modules/@dpc-sdp/ripple-form/index.vue
+++ b/node_modules/@dpc-sdp/ripple-form/index.vue
@@ -68,10 +68,13 @@ export default {
     submitFormOnClear: { type: Boolean, default: false },
     scrollToMessage: { type: Boolean, default: true },
     validateOnSubmit: { type: Boolean, default: true },
-    fullWidth: { type: Boolean, default: true }
+    fullWidth: { type: Boolean, default: true },
+    listenForClearForm: { type: Boolean, default: true }
   },
   mounted () {
-    RplFormEventBus.$on('clearform', this.clearForm)
+    if (this.listenForClearForm) {
+      RplFormEventBus.$on('clearform', this.clearForm)
+    }
 
     // TODO: We should abstract all future custom validators to a separate file and import them here.
     VueFormGenerator.validators.rplWordCount = function (value, field) {
